db/models.py

```diff
@@ -50,3 +59,48 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(default=datetime.now, blank=True)
```

Comment:
Using `datetime.now` for the `created_at` field can lead to timezone-related issues. It's better to use `django.utils.timezone.now` to ensure the time is aware of the timezone settings in your Django project.